sdk/test-utils/perfstress/src/tests.ts

```diff
@@ -24,6 +25,14 @@ export interface PerfStressTestConstructor<TOptions extends {} = {}> {
   new (): PerfStressTest<TOptions>;
 }
 
+/**
+ * Options provided for the test run.
+ * Includes both the default options as well as the custom options provided as part of the test.
+ */
+let _cachedParsedOptions:
+  | PerfStressOptionDictionary<DefaultPerfStressOptions>
+  | undefined = undefined;
```

Comment:
should this be a private member of the class instead? otherwise, I wonder how it will work if there're multiple instance of `PerfStressTest` at the same time.

Reply:
We only run one test at a time.
Multiple instances case occurs when we use the "parallel" option, all the multiple instances of `PerfStressTest` should take the exact same options for which caching helps.

Reply:
Oh please do not get me wrong, I am not asking you to do the support for different parallel instances in this PR or anywhere else any time soon, I just wanted to highlight the trade-off being made in this PR and make sure the side you took does not negatively affect future features. AFAICT, no other concerns from me so feel free to mark this thread as resolved.